packages/@aws-cdk/aws-eks/test/test.nodegroup.ts

```diff
@@ -598,6 +598,58 @@ export = {
     test.throws(() => cluster.addNodegroupCapacity('ng', { desiredSize: 2, minSize: 3 }), /Minimum capacity 3 can't be greater than desired size 2/);
     test.done();
   },
+  'can set minSize , maxSize and DesiredSize'(test: Test) {
+    // GIVEN
+    const { stack, vpc } = testFixture();
+    const cluster = new eks.Cluster(stack, 'Cluster', {
+      vpc,
+      defaultCapacity: 0,
+      version: CLUSTER_VERSION,
+    });
+    // WHEN
+    new eks.Nodegroup(stack, 'NodeGroup', {
+      cluster: cluster,
+      minSize: 2,
+      maxSize: 6,
+      desiredSize: 4,
+    });
+    // THEN
+    expect(stack).to(haveResourceLike('AWS::EKS::Nodegroup', {
+      ScalingConfig: {
+        MinSize: 2,
+        MaxSize: 6,
+        DesiredSize: 4,
+      },
+    },
+    ));
+    test.done();
+  },
+  'validation is not performed when using Tokens'(test: Test) {
+    // GIVEN
+    const { stack, vpc } = testFixture();
+    const cluster = new eks.Cluster(stack, 'Cluster', {
+      vpc,
+      defaultCapacity: 0,
+      version: CLUSTER_VERSION,
+    });
+    // WHEN
+    new eks.Nodegroup(stack, 'NodeGroup', {
+      cluster: cluster,
+      minSize: cdk.Lazy.number({ produce: () => 2 }),
+      maxSize: cdk.Lazy.number({ produce: () => 6 }),
+      desiredSize: cdk.Lazy.number({ produce: () => 4 }),
```

Comment:
Since the desired size is within the [min, max] bound, we can't really tell from this test that validation is not being performed. For all we know, it might have been performed and passed.

Reply:
Ah, got it, updated to show: 
```js
      minSize: cdk.Lazy.number({ produce: () => 5 }),
      maxSize: cdk.Lazy.number({ produce: () => 1 }),
      desiredSize: cdk.Lazy.number({ produce: () => 20 }),
```